gyp/pylib/gyp/generator/make.py

```diff
@@ -2469,16 +2483,34 @@ def CalculateMakefilePath(build_file, base_name):
         header_params.update({"link_commands": LINK_COMMANDS_ANDROID})
     elif flavor == "zos":
         copy_archive_arguments = "-fPR"
-        makedep_arguments = "-qmakedep=gcc"
+        CC = GetEnvironFallback(("CC_target", "CC"), "njsc")
+        makedep_arguments = "-MMD"
+        if CC == "clang":
+            CC_host = CC
+            CC_host = GetEnvironFallback(("CC_host", "CC"), "clang")
+            CXX_target = GetEnvironFallback(("CXX_target", "CXX"), "clang++")
+            CXX_host = GetEnvironFallback(("CXX_host", "CXX"), "clang++")
+        elif CC == "ibm-clang64":
+            CC_host = CC
+            CC_host = GetEnvironFallback(("CC_host", "CC"), "ibm-clang64")
```

Comment:
It looks like `CC_host` is assigned twice here as well similar to https://github.com/ibmruntimes/node-gyp/pull/5/files#r982693141